Dockerfile

```
@@ -0,0 +1,27 @@
+ARG RUBY_VERSION=${RUBY_VERSION:-2.7}
+FROM ruby:${RUBY_VERSION}-alpine
+
+RUN apk --update add --no-cache build-base bash && \
+  apk add git && \
+  apk add --no-cache libxml2 && \
+  apk add --no-cache libxml2-dev && \
+  apk add --no-cache sqlite-dev
+
```

If we update RubyGems in the Dockerfile then there's no need to add the ffi gem to the gemspec / Gemfile:
```suggestion
RUN gem update --system 3.3.22 --no-document && gem install bundler --no-document

```